### ISSUE TITLE
Parameter Editor - Save Parameter

### ISSUE THREAD

**Issue by BeamMeUp2018** [dreamos82/chrome_parameters#88]
It would be great to be able to save a parameter to add to a URL instead of having to type it in each time you use it.
If it was saved then you could just click the param to add. That would be perfect.

**Comment by dreamos82** [comment]
Yeah that is definetely a good idea, the only thing probably we need to discuss, is 
how to save the parameters. 

For example there are several way how to implement them:
1. Do we want the parameters saved by site? (but on the other hand, that means kind of implementing a bookmark mechanism, so technically speaking this features is kind of already existing in the browser, but probably an idea could be to create a specific bookmark folder to be watched by the extension, and used to read the parameters configuration)
2. Or the other option is to save parameters just plain, to be reused with any site, so you create your configuration of parameters and save it, and can reuse it with any website you know. 

These are the 2 way of implementing it that i acn think of, if you have any other idea, just add your comment.
